taccsite_cms/static/site_cms/css/src/site.css

```diff
@@ -17,7 +17,8 @@
 
 /* ELEMENTS */
 /* Bootstrap performs much of this */
-@import url("_imports/elements/figure.css");
+/* FP-1663: removed 'elements/figure.css' from @tacc/core-styles@0.6.0-beta.1 */
+/* @import url("_imports/elements/figure.css"); */
```

Comment:
Um... this means **both** that I should not merge https://github.com/TACC/Core-CMS/pull/505 before https://github.com/TACC/Core-CMS/pull/503 **and** that I should expect a merge conflict when I merge base branch to https://github.com/TACC/Core-CMS/pull/505 branch (after merging https://github.com/TACC/Core-CMS/pull/503).[^1]

[^1]: Alternatively, I could include all these fixes in one PR. Do that instead, yes/no?

Reply:
Thanks. When I get the third fix in ([FP-1665](https://jira.tacc.utexas.edu/browse/FP-1665)), I’ll verify whether combining the unmerged PRs into one is easier. Probably is.

<sup>The mini-releases have helped me keep things straight, but these fixes are so small that when combined with the disconnect from the actual changes, a PR each causes more work than should be necessary.</sup>

Reply:
Third fix will have its own PR (https://github.com/TACC/Core-CMS/pull/506), because it is not as simple as the first two and the first two have been approved (I will merge soon).